src/utils/tree.ts

```diff
@@ -0,0 +1,72 @@
+import { type ColorName, getColor } from "./color";
+
+export type TreeItemObject = {
+  /**
+   * Text of the item
+   */
+  text: string;
```

Comment:
Do you think you can continue to support nested tree objects as well?

Reply:
Yes, I can continue! Have you an image to view how does it should look like?

Reply:
Seems pretty cool! 
![image](https://github.com/unjs/consola/assets/45267552/6eabeaab-47ca-4dee-b497-54e7752b5aa2)